core/tauri-runtime-wry/src/lib.rs

```diff
@@ -3175,6 +3176,12 @@ fn create_webview<T: UserEvent>(
     }
   }
 
+  let label_c = label.clone();
+  if let Some(on_page_load_handler) = page_load_handler {
+    webview_builder =
+      webview_builder.with_on_load_handler(move |_, url| on_page_load_handler(&label_c, url));
+  }
+
```

Comment:
I don't love how this interfaces with the Wry API. I feel like there should be something better that could be done to get the Window object needed from the webview passed as a parameter instead of ignoring it and looking up the Window later by lable.

Reply:
This is better with the new API.